src/phoenix/frontend/prototype/main.go

```diff
@@ -45,31 +51,44 @@ func main() {
 
 	// run Frontend server
 	go frontend.ServeFrontend(feConfig)
-	<- feConfig.Ready
+	<-feConfig.Ready
+
+	if *useRand {
+		rand.Seed(time.Now().UnixNano())
+	} else {
+		// just some random number here
+		var randSeed int64 = 1111
+		rand.Seed(randSeed)
+	}
 
-	// TODO: randomize number of jobs or tasks
-	numTasks := 10
-	numJobs := 5
+	numTasks := *taskCount
+	numJobs := *jobCount
 
 	jobList := make([]*types.Job, 20)
-	var sumOfTaskTimes float32 = 0
+	var sumOfTaskTimes float64 = 0
 
 	// populate jobList
 	for i := 0; i < numJobs; i++ {
 		jobid := "job" + strconv.Itoa(i)
 		tasks := make([]types.Task, 0)
+
+		currTaskDuration := *meanDuration
```

Comment:
meanDuration is described as "job duration in second". Is that supposed to be used to set the Task Duration?

Reply:
yea, the description is not precise.